microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: ASSEMBLY
+[id="microshift-4-13-release-notes"]
+= {product-title} {product-version} release notes
+include::_attributes/attributes-microshift.adoc[]
+:context: release-notes
+
+toc::[]
+
+{product-title} provides developers and IT organizations with small-form-factor and edge computing, delivered as an application that customers can deploy on top of their managed {op-system-first} devices at the edge. Built on {op-system-first} and Kubernetes, {product-title} provides an efficient way to operate single-node clusters in low-resource edge environments.
+
+{product-title} is designed to make control plane restarts economical and be lifecycle-managed as a single unit by the operating system. Updates, roll-backs, and configuration changes consist of simply staging another version in parallel and then - without relying on a network - flipping to and from that version and restarting.
+
+[id="microshift-4-13-about-this-release"]
+== About this release
+
+The Red Hat build of {product-title} is now available as Technology Preview software. Features and known issues that pertain to {product-title} {product-version} are included in this topic. For more information about the support scope of Red Hat Technology Preview software, see link:https://access.redhat.com/support/offerings/
+
+//need messaging and link here
+
+[IMPORTANT]
+====
+Red Hat does not provide or support an update or upgrade path from the Technology Preview version to later versions of {product-title}. A new installation is necessary.
```

Comment:
I would get PX and support to look at statements like this.

Reply:
This is vetted already by PM and SME.